R/available.R

```diff
@@ -65,7 +65,11 @@ create <- function(name, ...) {
 #' @export
 suggest <- function(path = ".", title = NULL) {
   if (is.null(title)) {
-    title <- unname(desc::desc(path)$get("Title"))
```

Comment:
I think putting this line in a `tryCatch(..., error = function(e) NA)` block would be better, then execution would fall to line 71 with a more useful error.